pkg/network/network.go

```diff
@@ -390,6 +397,28 @@ func cnaoKNMStateObjects(operandNamespace string) ([]*unstructured.Unstructured,
 	return convertedObjects, nil
 }
 
+func cnaoIPAMControllerPasstObjects(operandNamespace string) ([]*unstructured.Unstructured, error) {
+	objects := []runtime.Object{
+		&appsv1.DaemonSet{
+			ObjectMeta: metav1.ObjectMeta{Namespace: operandNamespace, Name: "passt-binding-cni"},
```

Comment:
What about removing the passt NAD (on default ns)

Reply:
right